.circleci/config.yml

```
@@ -277,9 +277,12 @@ jobs:
             fi
             if ./Tests/scripts/is_ami.sh ;
               then
-                if [ -n "${NIGHTLY}" ] || [ -n "${INSTANCE_TESTS}" ] ;
+                if [ -n "${NIGHTLY}" ] ;
                   then
                     export IFRA_ENV_TYPE=Nightly # disable-secrets-detection
+                elif [ -n "${INSTANCE_TESTS}" ] ;
+                  then
+                    export IFRA_ENV_TYPE=Demisto PreGA # disable-secrets-detection
```

better safe than sorry
```suggestion
                    export IFRA_ENV_TYPE="Demisto PreGA" # disable-secrets-detection
```

---

:yaakovi: